File: 2418-sort-the-people/2418-sort-the-people.py
class Solution(object):
    def sortPeople(self, names, heights):
        arr = []
        
        for i in range(len(names)):
            arr.append([names[i], heights[i]])
        
        arr.sort(key=lambda x: x[1], reverse=True)
        
        result = []
        
        for i in range(len(arr)):
            result.append(arr[i][0])
        
        return result

File: 2418-sort-the-people/2418-sort-the-people.js
/**
 * @param {string[]} names
 * @param {number[]} heights
 * @return {string[]}
 */
var sortPeople = function(names, heights) {
    let arr=[]
    for(let i=0;i<names.length;i++){
        arr.push([names[i],heights[i]])
    }
    arr.sort((a,b)=>b[1]-a[1])
    let result=[]
    for(let i=0;i<arr.length;i++){
        result.push(arr[i][0])
    }
    return result
    
};